app/models/team-subscription.ts

```diff
@@ -0,0 +1,12 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+import type TeamModel from './team';
+
+export default class TeamSubscriptionModel extends Model {
+  @belongsTo('team', { async: false, inverse: 'subscriptions' }) declare team: TeamModel;
+  @attr('date') declare endedAt: Date;
```

Comment:
Is `endedAt` always a `Date`?

Reply:
This can be nil for ongoing subscriptions. Updated type as such.